test/runnable/issue22267.c

```diff
@@ -0,0 +1,19 @@
+typedef signed int int32_t;
+void exit(int);
+int printf(const char *fmt, ...);
+int32_t ret()
+{
+    int32_t init = (1 + 3);
+    return init;
+}
+
+int main()
+{
+    int32_t retVal = ret();
+    if(retVal != 4)
+    {
+        printf("ret() returned %s, expected 4\n", retVal);
+        exit(1);
```

Comment:
This could be redone into a `compilable` test using `_Static_assert`. These run much, much faster in the test suite.

Reply:
Can it? The actual meat of the test can be made into a compilable (i.e. it's a parser bug), but you can't call functions inside static assert, right?

Reply:
D's `_Static_assert` is more powerful than C's, so you can.

Reply:
So not compliant with the C spec 😘